diesel.toml

```diff
@@ -1,3 +1,4 @@
 [print_schema]
 file = "database/src/schema.rs"
```

Comment:
if this is the schema file, then how come we're deleting it, i'm a bit confused…

Reply:
`database/src/schema.rs` is generated by `diesel print-schema` inside `scripts/setup`.  (This branch adds some scripts to setup and run the server; if/when this is merged, those scripts will probably become part of the standard setup/run procedure.)

Reply:
iirc it's diesel best practice to check schema.rs into source control (so you don't have to print-schema when deploying the app For Realz). don't quote me on that though

Reply:
although that gets done as part of migrations so extremely :woman_shrugging:

Reply:
I think that makes sense in situations where you're pushing source code to a bunch of app servers (which is the classic Web application deployment model), but I don't think it matters much when you're building a binary and deploying that to a bunch of app servers or (more common in the SQLite case) building a binary and installing it on many machines, since you won't be recompiling on the destination machine in either case.

That said, if there arises some benefit to checking in `schema.rs`, it doesn't seem like it would be a difficult change to implement.

Reply:
Yeah, we have it checked in right now and i think that's the only reason to keep it around atm. Pulling it out is totally fine :purple_heart: